src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+
+    public CinemaHall() {
+    }
```

Comment:
Don't create a default constructor when it is not needed. Since there are no other constructors defined and no explicit requirements for a default constructor, it should be removed.